persistence/src/main/resources/db/migration/data/integration/V1_12_2__AddStateFlow.sql

```diff
@@ -0,0 +1,64 @@
+INSERT INTO state_flow (from_id,to_id) VALUES (1,2);
```

Comment:
Like for the state values, this SQL migration script should be placed into the schema folder.